conda_smithy/templates/circle.yml.tmpl

```diff
@@ -28,7 +28,7 @@ test:
     # Run, test and (if we have a BINSTAR_TOKEN) upload the distributions.
     - ./ci_support/run_docker_build.sh
 {% else %}
-    # In case the skip fails for some reason, do a fast finish.
+    # The Circle-CI build should not be active but if this is not true for some reason, do a fast finish.
```

Comment:
Very minor grammar note. We should probably have a `,` before `but`. Though it isn't a big deal to me either way.